### PR TITLE
[PSM Interop] used fixed names like latest, oldest to maintain history

### PR DESCRIPTION
**Pull request by sanjaypujare**

### CODE REVIEW

**Comment by sanjaypujare** [comment]
Test runs

- https://fusion2/8561919c-ae38-4386-ae2a-9260fc7a906a (xbranch and completed)
- https://fusion2/86d1c3bc-0ff8-4042-9327-3950fd15a5bd (xlang and running)

File: tools/internal_ci/linux/grpc_xds_k8s_run_xtest.sh [hunk 1]
```diff
@@ -87,9 +89,10 @@ run_test() {
   local server_image_name_tag="${server_image_name}:${server_branch}"
   local client_image_name_tag="${client_image_name}:${client_branch}"
 
-  local out_dir="${TEST_XML_OUTPUT_DIR}/${client_branch}-${server_branch}/${client_lang}-${server_lang}"
+  local out_dir="${TEST_XML_OUTPUT_DIR}/${client_branch_fixed}-${server_branch_fixed}/${client_lang}-${server_lang}"
   mkdir -pv "${out_dir}"
   set -x
+  echo "Client branch=" "${client_branch}" ", Server branch=" "${server_branch}" > ${out_dir}/sponge_log.log
```
Comment (sergiitk) [inline tools/internal_ci/linux/grpc_xds_k8s_run_xtest.sh:95]:
```suggestion
  echo "Client branch='${client_branch}', Server branch='${server_branch}'" > ${out_dir}/sponge_log.log
```

Not sure what you meant by `branch=" "${client_branch}" "`, the output looks like this:
```
Client branch= master , Server branch= v1.56.x
```


Did you mean to make it something like this?

```
Client branch='master', Server branch='v1.56.x'
```

**Review by sergiitk — COMMENTED** [review]
LGTM, but wait for the xlang to see how the results look like.

**Comment by sanjaypujare** [comment]
> LGTM, but let's wait for the xlang to see how the results look like.

xlang done. PTAL at http://fusion2/86d1c3bc-0ff8-4042-9327-3950fd15a5bd

**Pull request merged**